src/arch/aarch64/console.rs

```diff
@@ -0,0 +1,60 @@
+use core::ptr::NonNull;
+
+use hermit_dtb::Dtb;
+
+pub struct Console {
+	stdout: NonNull<u8>,
+}
+
+fn stdout() -> u32 {
```

Comment:
I found this fn name a little confusing. Maybe something like `get_stdout` or `stdout_addr` would be more self-explanatory. A doc-comment might also help.